server/config.yaml.dist

```diff
@@ -4,7 +4,7 @@
 # shown in the website title and on the front page
 name: szurubooru
 # full url to the homepage of this szurubooru site, with no trailing slash
-domain: # example: http://example.com
+domain: localhost # example: http://example.com
```

Comment:
This is an optional parameter and should be blank by default. It is used in the code for sending password reset emails.